apps/frontend/app/components/media.tsx

```diff
@@ -601,12 +601,9 @@ export const MetadataDisplayItem = (props: {
 					<Group gap={4}>
 						<IconStarFilled size={12} style={{ color: "#EBE600FF" }} />
 						<Text c="white" size="xs" fw="bold" pr={4}>
-							{match(userPreferences.general.reviewScale)
-								.with(UserReviewScale.OutOfFive, () =>
-									Number.parseFloat(averageRating.toString()).toFixed(1),
-								)
-								.with(UserReviewScale.OutOfHundred, () => averageRating)
-								.exhaustive()}
+							{Number(averageRating) % 1 === 0
```

Comment:
Can you also try changing your review preferences to `OutOFive` and see that it works as expected? If you rate something 80%, it should show 4 stars.

Reply:
The behavior is the same as before if you use OutOfFive. This is without the patch

![image](https://github.com/user-attachments/assets/cce5a52f-29f8-4f8e-a567-5c4c575e4fe1)


This is with the patch

![image](https://github.com/user-attachments/assets/e63b0f60-b484-4641-83cc-08579b53ffa9)

note the 4.0 being dropped for a 4.